pkg/models/errors.go

```diff
@@ -61,6 +61,7 @@ var (
 	ErrPrivateLinkSupportedOnlyForSingleDC      = errors.New("private link is only supported for a single data centre")
 	ErrPrivateLinkSupportedOnlyForAWS           = errors.New("private link is supported only for an AWS cloud provider")
 	ErrImmutableSpec                            = errors.New("resource specification is immutable")
+	ErrUnsupportedBackupClusterKind             = errors.New("Backups for provided cluster kind are not supported")
```

Comment:
Backups for provided cluster kind are not supported -> backups for provided cluster kind are not supported

Reply:
fixed